keyboards/whitefox/keymaps/constexpr/config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you remove the first three lines and the `#endif` at the end and use just `#pragma once`

Eg, remove this:
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "../../config.h"
```
And the `#endif` at the end.